### ISSUE TITLE
test_scaled test failure using python 2.7.13

### ISSUE THREAD

**Issue by andyprata** [Unidata/netcdf4-python#642]
Hi, I'm getting the following test failure (note: I'm using a macports installation of python):
```
$ python run_all.py

not running tst_unicode3.py ...

netcdf4-python version: 1.2.6
HDF5 lib version:       1.8.18
netcdf lib version:     4.4.1
numpy version           1.12.1
..............................F............................................................
======================================================================
FAIL: test_scaled (tst_masked5.VectorMissingValues)
Testing auto-conversion of masked arrays
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/andyprata/netcdf4-python-master/test/tst_masked5.py", line 44, in test_scaled
    self.assertTrue(isinstance(v[:], ma.core.MaskedArray))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 91 tests in 27.357s

FAILED (failures=1) 
```
Any thoughts on how I can resolve this?  Thanks in advance.

**Comment by jswhit** [comment]
Looks like you have a mixture of version 1.2.6 and 1.2.7 installed.  From the Changelog:

```
 version 1.2.7 (tag v1.2.7rel)
==============================
 * fix for issue #624 (error in conversion to masked array when variable slice
   returns a scalar). This is a regression introduced in 1.2.5 associated
   with support for vector missing_values. Test (tst_masked5.py) added for
vector missing_values.
```

so tst_masked5.py was only added in 1.2.7, specifically to test for a regression introduced in 1.2.6.  Looks like you are running the test with version 1.2.6.  If you upgrade everything to 1.2.7, all should be well.

**Comment by andyprata** [comment]
Thanks.  The following solved my issue:
```
sudo pip install netcdf4 --upgrade
```
